api/napi/src/interpreter/window.rs

```diff
@@ -18,13 +21,16 @@ impl From<WindowAdapterRc> for JsWindow {
 
 #[napi]
 impl JsWindow {
+    /// @hidden
     #[napi(constructor)]
     pub fn new() -> napi::Result<Self> {
         Err(napi::Error::from_reason(
             "Window can only be created by using a Component.".to_string(),
         ))
     }
 
+    /// Shows the window on the screen. An additional strong reference on the
+    /// associated component is maintained while the window is visible.
```

Comment:
I wonder if the strong reference aspect here needs documenting. After all, all references in the JS bindings are strong. So if you create a component instance and "only" access it from within a closure, it'll still be around.

```
let blah = new compiledModule.AppWindow();

blah.mycallback.setHandler(() => {
    blah.foo = 42; // Boom, blah is null here
});

blah.run();
blah = null;
```

And if `blah` is not set to null, it won't ever be collected because the closure still references it and we keep a strong reference to the closure.

@ogoffart what do you think?